src/components/atoms/SpecialServiceCard/SpecialServiceCard.module.scss

```diff
@@ -0,0 +1,24 @@
+@use "sass:map";
+@import "../../../foundations/sass/all";
+
+.service-card {
+  @include center;
+  flex-direction: column;
+  
+  gap: 15px;
+  font-family: $playfair;
+
+  &__title {
+    color: $special-service-heading-color;
+    @include center;
+    font-size: $fs-xxl;
+   @include margin-bottom($spacing);
+    font-weight: $fw-bold;
+  }
+
+  &__description {
+    color: $special-service-description-color;
+    font-size: $fs-xl;
```

Comment:
It should be $fs-md. Please visit our figma template at
https://www.figma.com/design/xiAbsYNAyBYPH5pfPUkIEX/Restaurant-website-template-(Community)?node-id=42-834&node-type=FRAME&t=dzwsbcPpXgGU4aBT-0